### PR TITLE
Add 1.13.2 Changelog, update rc image

### PR DESCRIPTION
**Pull request by amacaskill**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design

 /kind documentation

> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
Add 1.13.2 Changelog and update rc image to 1.13.2-rc1

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.13.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
+# v1.13.2 - Changelog since v1.13.1
+
+## Changes by Kind
+
+### Bug
+
+- Fix error when no compute endpoint is passed ([#1622](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1622), [@pwschuurman](https://github.com/Sneha-at))
```
Comment (pwschuurman) [inline CHANGELOG/CHANGELOG-1.13.md:7]: This should be `Sneha-at` in the square brackets

Comment (amacaskill) [reply CHANGELOG/CHANGELOG-1.13.md]: Thanks! Done

**Pull request merged**